docs/src/pages/comparing-astro-vs-other-tools.md

```diff
@@ -0,0 +1,136 @@
+
+#### Comparing Docusaurus vs. Astro Performance
+
+In almost all cases, a website built with Astro will load significantly faster than a website built with Docusaurus. This is because Astro uses Partial Hydration and Island Architecture to only load the bare minimum amount of JavaScript in the browser. Docusaurus websites don't support this, and instead load your entire JavaScript application in the browser. The result is a slower page load and worse Lighthouse scores than a similar website built with Astro. 
+
+Case Study: [docusaurus.io/docs](https://docusaurus.io/docs) (built with Docusaurus) loads **238kb** of JavaScript on first load, and scores a Lighthouse performance score of [26 out of 100](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fdocusaurus.io%2Fdocs). This website, (built with Astro) loads only **9.3kb** of JavaScript (96% less) and scores a Lighthouse performance score of [94 out of 100](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fdocs.astro.build%2Fgetting-started). 
+
+
+## Eleventy vs. Astro
+
+[Eleventy](https://www.11ty.dev/) is a popular static site builder, powered by Node.js. It is very similar to Astro, and conceptually aligned with Astro's "minimize client-side JavaScript" approach to web development.
+
+Eleventy supports several older [templating languages](https://www.11ty.dev/docs/languages/) for HTML: Nunjucks, Liquid, Pug, EJS, and others. Astro lets you write HTML using modern UI component libraries (React, Preact, Vue, Svelte) or a custom component syntax that is similar to HTML + JSX. Eleventy does not currently support using modern UI component libraries for HTML templating.
+
+Eleventy offers "zero client-side JavaScript" by not including any JavaScript on your site by default. By making JavaScript opt-in, Eleventy sites can be written with little to no JavaScript. However, when you do need client-side JavaScript, it's up to you to set up your own build pipeline. This can be time consuming, and is only as optimized as what you can build yourself with Webpack, Snowpack, or Vite.
+
+By contrast, Astro has built-in support for Partial Hydration and Island Architecture to only load the bare minimum amount of JavaScript in the browser. In production, Astro automatically optimizes your included component JavaSript bundles for you. This gives you built-in support to include individual JavaScript components on the page, without any effort on your part.
+
+#### Comparing Eleventy vs. Astro Performance
+
+Eleventy has few opinions about what you build, so it's difficult to compare generally. Because Eleventy includes zero JavaScript by default, it is possible for an Eleventy website to be as fast as an Astro website. However, because you are responsible for building your own build pipeline for any JavaScript that you include on the page, most Eleventy websites will not be as well-optimized as Astro websites.
```

Comment:
> Eleventy has few **opinions** about what you build

Not sure if it's really meant as opinions or options

Reply:
I parsed it as "11ty doesn't care a lot about what you build", though I do agree it's a little confusing.

Reply:
fixed!